globaldog.py

```diff
@@ -27,6 +27,8 @@ def handleErrors(datadogResponse):
     "app_key": os.environ.get("DATADOG_APP_KEY"),
```

Comment:
```python
-print(datadogCredentials)
+# Remove or mask sensitive credentials information rather than printing it.
+# For example, if debugging is necessary, consider logging the existence of credentials without exposing their value:
+# logger.debug("Datadog credentials loaded")  # Using a secure logging method with proper log level.
```

Printing sensitive credentials is a security risk that can expose sensitive data.